src/backend/types/gmaclennan_zip-fs.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for @gmaclennan/zip-fs 1.2.0
+// Project: https://github.com/gmaclennan/zip-fs
+// TypeScript Version: 4.2
+
+/// <reference types="node" />
```

Comment:
What does this line do?

Reply:
I don't fully understand these ["triple-slash directives"](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-path-) in Typescript, but I am copying this from other modules: it tells Typescript that the types reference types available in node.